xinference/model/llm/llm_family.py

```diff
@@ -990,3 +1002,36 @@ def match_llm_cls(
             if cls.match(family, llm_spec, quantization):
                 return cls
     return None
+
+
+def check_engine_by_spec_parameters(
+    model_engine: str,
+    model_name: str,
+    model_format: str,
+    model_size_in_billions: Union[str, int],
+    quantization: str,
+) -> Optional[Type[LLM]]:
+    if model_name not in LLM_ENGINES:
+        print(f"Model {model_name} not found")
+        logger.debug(f"Model {model_name} not found")
```

Comment:
All the log information here should be 
`Model {model_name} with format {model_format}, size {model_size_in_billions} and quantization {quantization} cannot be run on engine {model_engine}.`